plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -15,14 +17,19 @@ import ServiceTree from "../../structs/ServiceTree";
 
 // This needs to be at least equal to @modal-animation-duration
 const REDIRECT_DELAY = 300;
-const METHODS_TO_BIND = ["handleRightButtonClick"];
+const METHODS_TO_BIND = [
+  "handleRightButtonClick",
+  "handleChangeInputFieldDestroy"
+];
 
 class ServiceDestroyModal extends React.Component {
   constructor() {
     super(...arguments);
 
     this.state = {
-      errorMsg: null
+      errorMsg: null,
+      inputFieldDestroy: "",
```

Comment:
Suggestion `serviceNameConfirmationValue`?
`inputFieldDestroy` implies it's a ref to the input field, but default value is an empty string.